code/src/components css/messageList.css

```diff
@@ -0,0 +1,40 @@
+.message {
+  background: #fff2e2;
+  border: 2px solid black;
+  box-shadow: 4px 4px black;
+  width: 70vw;
+  height: auto;
+  margin: 20px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  font-family: "Courier New", Courier, monospace;
+  padding: 20px;
+  overflow-wrap: break-word;
+}
+.text {
+  font-family: "Courier New", Courier, monospace;
+  font-size: 20px;
+  margin: 0;
+}
+.message-time {
+  display: block;
+  font-family: "Courier New", Courier, monospace;
```

Comment:
I see you've added the font family in a lot of classes. Perhaps this could be in shared class/tag (like body) so you don't need to repeat the same code. This is also good if you sometime in the future want to change the font, then you only need to do it at one place. ☺️